### DISCUSSION TITLE
Older versions of GrandOrgue

### DISCUSSION THREAD

**Discussion by BuzzAgain** [GrandOrgue/grandorgue#1639] [Help]
Is it possible to get GrandOrgue to work in macOS High Sierra. I'd like to run GrandOrgue on my slave computer which is very old MacBook Pro. I tried version: GrandOrgue-0.3.1.2321-Darwin.dmg but it just loads and fails to render any panels.

**Comment by oleg68** [comment]
[GrandOrgue 3.11.2](https://github.com/GrandOrgue/grandorgue/releases/tag/3.11.2-1) is the last version that is capable to run in MacOS 10.15.
